### PR TITLE
Dynamic: created-aside-on-npm-install-save

### PR DESCRIPTION
**Pull request by MayaGans**
Before we load libraries (in the Hello Bottom example) am I correct that we need to have them installed locally? I created an aside here to introduce the reader to saving modules. 

I also added the type to /src/dynamic/hello-bottom/index.html to make sure once I saved the modules I'd see "Rendered by React" but I still do not (see issue #10 )

### CODE REVIEW

File: dynamic.tex [hunk 1]
```diff
@@ -426,6 +426,23 @@ \section{Multiple Files}\label{s:dynamic-multiple}
...
 
+
+\begin{aside}{Installing React and ReactDOM}
+
+Now that we've made a project folder using \texttt{npm init},
+and we have a \texttt{project.json} file,
+we can install our first modules:
+
+\begin{minted}{shell}
+install --save react
```
Comment (gvwilson) [inline dynamic.tex]: isn't this `npm install` ?

Comment (gvwilson) [reply dynamic.tex]: and do we need `--save` explicitly? I think that's the default?

File: src/dynamic/hello-bottom/#index.html# [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
```
Comment (gvwilson) [inline src/dynamic/hello-bottom/#index.html#]: This file probably shouldn't be in the commit - with the leading/trailing `#` it looks like it's some kind of editor backup?

**Comment by MayaGans** [comment]
I added a sentence so that the reader knows moving forward they'll need to install the libraries whenever they see `require` (rather than adding `install x` before each code chunk) but I'm not attached to the way its phrased here.

File: dynamic.tex [hunk 3]
```diff
@@ -426,6 +426,23 @@ \section{Multiple Files}\label{s:dynamic-multiple}
 </html>
 \end{minted}
 
+
+\begin{aside}{Installing React and ReactDOM}
+
+Now that we've made a project folder using \texttt{npm init},
+and we have a \texttt{project.json} file,
```
Comment (gvwilson) [inline dynamic.tex]: "which created a project.json file" maybe, to make it clear that `npm init` does the creating?

File: dynamic.tex [hunk 4]
```diff
@@ -426,6 +426,23 @@ \section{Multiple Files}\label{s:dynamic-multiple}
...
+we can install our first modules:
```
Comment (gvwilson) [inline dynamic.tex]: do we tell people anywhere that they have to go into the newly-created project directory before running `npm install`?

File: dynamic.tex [hunk 5]
```diff
@@ -426,6 +426,23 @@ \section{Multiple Files}\label{s:dynamic-multiple}
...
+
+\begin{minted}{shell}
+npm install react
+npm install reactDOM
+\end{minted}
+
+Any code chunks beginning with the function \texttt{require},
```
Comment (gvwilson) [inline dynamic.tex]: Um... code chunks? In what? Do you mean calls to `require` in example code snippets?

**Pull request merged**